frontend/src/views/ImageDetailsPage/components/FeatureTiles/LandmarksMapTile/LandmarksMapTile.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import { FeatureTemplate } from 'assets/styles/style';
+import { fetchMapAPIConfig } from 'API';
+
+export const LandmarksMapTile = ({ landmarks }) => {
+  const [mapUrl, setMapUrl] = useState('');
+
+  useEffect(() => {
+    const getMarks = async (landmarks) => {
+      let mapURL = 'https://maps.googleapis.com/maps/api/staticmap?&zoom=12&size=250x250&maptype=roadmap&markers=color:red%7Clabel:A%7C';
+      const mapKey = await fetchMapAPIConfig();
+      let x = landmarks?.map((obj) => {
+        return (x = obj.latitude);
```

Comment:
Why not just `x = landmarks[0].latitude`? Assuming `landmarks` always have at least one element.